test/e2e/network/service.go

```diff
@@ -3998,9 +3998,15 @@ var _ = SIGDescribe("SCTP [Feature:SCTP] [LinuxOnly]", func() {
 			err := cs.CoreV1().Pods(f.Namespace.Name).Delete(context.TODO(), podName, metav1.DeleteOptions{})
 			framework.ExpectNoError(err, "failed to delete pod: %s in namespace: %s", podName, f.Namespace.Name)
 		}()
+		// wait until host port manager syncs rules
+		// TODO: make this more reliable
+		time.Sleep(5 * time.Second)
```

Comment:
We just need to wait for the pod to become Ready; that implies that the CNI plugin has done its work, which implies that the hostports should be set up. There's some framework method that can be used for this.

Reply:
CreateSync() already waits it is running 

https://github.com/kubernetes/kubernetes/blob/3f585835d0a38462bb928454245ec01e834e53a0/test/e2e/framework/pods.go#L99-L108

Reply:
If the pod has an IP then shouldn't it be guaranteed that the hostport manager already ran?

Reply:
maybe lagging programming the iptables rules?
let me add an active polling for the iptables check